README.md

```diff
@@ -17,7 +17,7 @@ To get started with APM, see our [Quick start guide](https://www.elastic.co/guid
 
 ### Requirements
 
-* [Go][golang-download] 1.22.x
+* [Go][golang-download]
```

Comment:
What do you think about mentioning the `.go-version` file here without an explicit reference to the version?

```suggestion
* [Go][golang-download] (prefer installing the version specified in `.go-version`)
```